src/modules/task.js

```diff
@@ -0,0 +1,169 @@
+import listItem from './listofItems.js';
+
+export default class TaskList {
+  constructor() {
+    this.listObj = JSON.parse(localStorage.getItem('Tasks')) || [];
+  }
+
+    self = this;
+
+    i = 0;
+
+    add(description, completed = false, index = (this.i += 1)) {
+      this.listObj.push({ description, completed, index });
+      this.populateLocalStorage();
+    }
+
+    findIndex() {
+      let k = 0;
+      this.listObj.forEach((elem) => {
+        k += 1;
+        elem.index = k;
+      });
+    }
+
+    removeTask(checkbox) {
+      const checkboxId = checkbox.target;
+      if (checkboxId.checked) {
+        document
+          .getElementById(`label-text ${checkboxId.id}`)
+          .classList.add('checked');
+        this.listObj[checkboxId.id - 1].completed = true;
```

Comment:
- [OPTIONAL ] As the previous reviewer mentioned Kindly make sure that when the Clear all completed button is clicked, it should only delete the `checked` task but in your app, it is removing all` unchecked` tasks.